spec/dataframe_spec.rb

```diff
@@ -423,7 +423,7 @@
     end
   end
 
-  context "#[]" do
+  context "#vector[]" do
```

Comment:
The `df.vector[]` syntax has been deprecated. See https://github.com/v0dro/daru/pull/136/files